lib/Doctrine/DBAL/Migrations/Configuration/Configuration.php

```diff
@@ -742,7 +738,7 @@ public function getMigrationsToExecute($direction, $to)
         }
 
         if ($direction === Version::DIRECTION_DOWN) {
-            if (count($this->migrations)) {
+            if ($this->migrations) {
```

Comment:
https://github.com/ReenExeStudyTools/PHPStudy/blob/master/tests/Basic/BooleanTest.php
All filled array is `true` (exception when implements \\Countable)
In this project sometime use `!empty($array)` or `count($array)` -> but enough if ($array) {}

See please this link:
http://php.net/manual/en/types.comparisons.php
Comparisons of $x with PHP functions:
Expression	boolean : if($x)
$x = array();	FALSE
$x = array('a', 'b'); TRUE